src/jdk.jdwp.agent/share/native/libjdwp/eventHelper.c

```diff
@@ -605,6 +624,35 @@ handleReportVMInitCommand(JNIEnv* env, ReportVMInitCommand *command)
     /* Why aren't we tossing this: tossGlobalRef(env, &(command->thread)); */
 }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+static void
+handleReportVMRestoreCommand(JNIEnv* env, ReportVMRestoreCommand *command)
```

Comment:
Space before, but not after `*` (this line isn't even self-consistent).

Reply:
Fixed in https://github.com/ibmruntimes/openj9-openjdk-jdk/commit/3b8e11025cb78b82767c77f772263c92cd01ad2a